app/views/docs/oauth-providers/zoom.md

```diff
@@ -0,0 +1,342 @@
+# Zoom provider
+
+Appwrite allows you to authenticate users using their Zoom account through Zoom OAuth2 provider. OAuth authentication is a great way to reduce friction for your users and increase user conversion by simplifying the signup process. You can learn more about Appwrite's other OAuth2 providers [here](placeholder link). 
+
+## Enabling the Zoom provider
+Before you can use Zoom to authenticate users, you need to enable the provider in your Appwrite console.
+1. Navigate to your Appwrite project
+2. Navigate to **Auth** > **Settings**
+3. Find and open the OAuth provider
+4. In the **Zoom OAuth2 Settings** modal, use the toggle to enable the provider
+
+Don't close this modal, we'll need to create a Zoom OAuth app in Zoom App Marketplace to complete this form.
+
+## Creating a Zoom app for OAuth
+To establish Zoom OAuth integration with Appwrite, start by creating a Zoom OAuth app via your [Zoom App Marketplace Dashboard](https://marketplace.zoom.us/user/build). In the navigation bar, access the **Build App** menu and opt for an OAuth app. Assign a suitable name to your app and proceed with its creation. In the **App Credentials** section, When prompted to provide a **Redirect URI**, , provide the **URI** found in the **Zoom OAuth2 Settings** modal from your Appwrite console. Also include your domain name in the designated allow list. In the **Information** section, add basic app information and developer contact information. Add all necessary scopes for your app in **Scopes** section. 
+
+After you've created your Zoom OAuth app, you can head back to your Appwrite console to complete the form in the **Zoom OAuth2 Settings** modal.
+- You can find the **Client ID** in your Zoom OAuth app credentials and provide this in the **App ID** field in the **Zoom OAuth2 Settings** modal from the Appwrite console.
```

Comment:
Find the Client ID in your Zoom OAuth app credentials. Provide this in the Appwrite console's Zoom OAuth2 Settings modal, in the App ID field.

Reply:
Thanks! @Haimantika changing it accordingly.